src/components/Skeletons/TableListItemSkeleton.tsx

```diff
@@ -1,17 +1,103 @@
 import React from 'react';
-import {Rect} from 'react-native-svg';
+import {Circle, Rect} from 'react-native-svg';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import CONST from '@src/CONST';
 import ItemListSkeletonView from './ItemListSkeletonView';
 
 type TableListItemSkeletonProps = {
     shouldAnimate?: boolean;
     fixedNumItems?: number;
 };
 
+const circleRadius = 8;
+const padding = 12;
```

Comment:
I think the padding on mobile for these cards is 12 vertical, and 16 horizontal.

Reply:
@shawnborton Can you share the design about the "padding" you mentioned above?

Reply:
This is what the padding should look like:
![CleanShot 2024-06-14 at 17 28 36@2x](https://github.com/Expensify/App/assets/2319350/205beafd-e67e-4109-a30d-aa4ce04761c8)

Reply:
And there should be an 8px gap between the content at the top of the row and the content at the bottom:
![CleanShot 2024-06-14 at 17 29 07@2x](https://github.com/Expensify/App/assets/2319350/12b9ba6f-2813-4c65-9d5d-86501b28045b)